.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
 env:
   matrix:
     # django
-    - DJANGO_VERSION=1.8 DJANGO_SETTINGS_MODULE="contributr.settings"
+    - DJANGO_VERSION=1.8 DJANGO_SETTINGS_MODULE="contributr.settings.testing"
```

Comment:
Can someone verify if this is the right way to point it to the settings folder? the settings folder is a module, I don't know why it won't see it.